src/dom.d.ts

```diff
@@ -1593,12 +1593,17 @@ export type AnchorAriaRoles =
 			role?: Signalish<AriaRole | undefined>;
 	  };
 
-export type AnchorHTMLAttributes<T extends EventTarget = HTMLAnchorElement> = Omit<
+export type AccessibleAnchorHTMLAttributes<T extends EventTarget = HTMLAnchorElement> = Omit<
 	PartialAnchorHTMLAttributes<T>,
 	'role'
 > &
 	AnchorAriaRoles;
 
+export interface AnchorHTMLAttributes<T extends EventTarget = HTMLAnchorElement> extends PartialAnchorHTMLAttributes<T> {
+	href?: Signalish<string | undefined>;
+	role?: Signalish<AriaRole | undefined>;
+}
```

Comment:
This is all that the PR does; makes plain & wide interfaces to compliment the specific accessible types.